core/src/plugin/bedrock.ts

```diff
@@ -0,0 +1,115 @@
+import { BedrockRuntimeClient, InvokeModelCommand } from '@aws-sdk/client-bedrock-runtime';
+import { Content } from "../content.js";
+import { isDefined } from "../util.js";
+import { Message, Summary } from "./index.js";
+
+export const DEFAULT_MODEL = 'anthropic.claude-v2';
+
+export function buildClaude2Prompt(messages: Message[]) {
+  let output = '';
+  let prevRole = "";
+
+  const roleAdjustedMsgs = messages.map(msg => ({ ...msg, role: msg.role === 'assistant' ? 'Assistant' : 'Human' }))
+
+  roleAdjustedMsgs.forEach(msg => {
+    if (msg.role !== prevRole) {
+      output += '\n\n' + msg.role + ': ';
+      prevRole = msg.role;
+    }
+    else {
+      output += '\n';
+    }
+
+    output += msg.content;
+  });
+
+  return output + '\n\nAssistant:';
+}
+
+export async function generate(
+  c: Content,
+  {
+    model = DEFAULT_MODEL,
+  }: { model: string; }
+): Promise<{ message: string; debug: unknown }> {
+  const bedrock = new BedrockRuntimeClient({});
+  let messages = c.meta?.messages ?? [];
+  if (messages.length < 2) {
+    throw new Error("Not enough messages");
+  }
+  if (messages.at(-1)?.role == "assistant") {
+    messages = messages.slice(0, -1);
+  }
+  console.log(
```

Comment:
You probably copy/pasted this from the other example? Maybe we should pull the `generate` logging up a level.